onnxruntime/core/providers/dml/DmlExecutionProvider/src/Operators/DmlOperator.cpp

```diff
@@ -111,6 +111,38 @@ namespace Dml
         }
     }
 
+    void DmlOperator::SetDmlOperatorDesc(
```

Comment:
>void DmlOperator::SetDmlOperatorDesc( [](http://example.com/codeflow?start=4&length=37)

To be consistent with other overloads, I believe this should patch the graph according to m_kernelInputIndices and m_kernelIOutputIndices.  Those members can then remain private

Reply:
Consider passing the graph description by rvalue reference to avoid copies while patching it

Reply:
Or just implement the remapping entirely in the derived class, then make this method check that the mappings have been set to identity

Reply:
I made m_kernelInputIndices and m_kernelOutputIndices private again. For this particular case the remapping between OnnInputIndices and DmlInputIndices is identity, but it won't be the case with all operators like BatchNorm. 
So, I think we should not have check that remapping should be identity in the base class and rely on the derived class for that.